src/views/routes/users.js

```diff
@@ -18,17 +18,17 @@ router.post('/register', async (req, res) => {
 router.post('/login', async (req, res) => {
   try {
     const { email, password } = req.body;
-    const { token, userExists } = await generateToken(email, password);
+    const token = await generateToken(email, password);
 
-    if (!userExists) {
-      return responseHandler.handleError(res, 404, 'The user is not registered');
-    }
-    if (!token) {
-      return responseHandler.handleError(res, 401, 'Password incorrect');
-    }
     responseHandler.handleSuccess(res, 200, token);
   } catch (error) {
-    responseHandler.handleError(res, 500);
+    const { message } = error;
+
+    if (message.includes('The user is not registered') || message.includes('Password is incorrect')) {
```

Comment:
This is a good start but it feels a bit rushed. Take a look at the example of how we have the same thing set up in the blog. It promotes reuse and easier management of these constants by abstracting them to a different file and having a consistent structure of how they are accessed. https://bitbucket.t4g.com/projects/MSD/repos/demo-blog---backend/browse/src/views/routes/auth-login.js

Don't rush through it. Remember every part of the application is important, even if you're focusing on dev-ops right now.